File: Competicoes Diversas/2020/Rinha de Calouro/Pacotes/K/solver/k.cpp
#include <bits/stdc++.h>
using namespace std;

typedef long long ll;

int main(){
    int casos;
    cin >> casos;
    while(casos--){
        ll num;
        cin >> num;

        bool flag = true;
        for (int i = 2; i <= sqrt(num); i++){
            if (num % i == 0){
                flag = false;
                cout << "ei Kaqui, esse numero nao eh primo"<< endl;
                break;
            }
        }

        if(flag){
            cout << (int) log10(num)+1 << endl;
        }
    }
}

File: Competicoes Diversas/2020/Rinha de Calouro/Pacotes/K/solver/k.py
import math

casos = int(input())

for i in range(casos):
    num = int(input())

    for j in range(2, int(math.sqrt(num))+1):
        if num % j == 0:
            print("ei Kaqui, esse numero nao eh primo")
            break
    else:
        aux = int(math.log(num, 10)) + 1
        print(aux)
